doc/source/examples.rst

```diff
@@ -199,3 +199,24 @@ as the pull request gets updated with new commits.
           - base=master
         actions:
           dismiss_reviews:
+
+Require All Requested Reviews to Be Approved
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If all requested reviews have been approved, then the number of
+``review-requested`` and the number of ``changes-requested-reviews-by`` will
+both be 0.
+
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: remove outdated reviews
+        conditions:
+          - "#review-requested=0"
+          - "#changes-requested-reviews-by=0"
```

Comment:
Hum I think that'd merge even with 0 review since if you did not request any review `#review-requested=0` and since nobody asked for a change `#changes-requested-reviews-by=0` is true.

Reply:
That's correct, it would. Though I think for this example, that makes sense – it's only verifying that all requested reviews have been approved. It doesn't make any assumptions about whether a review is required in the first place or not.

Reply:
Yeah ok, I thought it was confusing but re-reading it seems ok to me. :)